selfdrive/ui/layouts/home.py

```diff
@@ -139,7 +139,7 @@ def _render_header(self):
       highlight_color = rl.Color(255, 140, 40, 255) if self.current_state == HomeLayoutState.UPDATE else rl.Color(255, 102, 0, 255)
       rl.draw_rectangle_rounded(self.update_notif_rect, 0.3, 10, highlight_color)
 
-      text = "UPDATE"
+      text = tr("UPDATE")
```

Comment:
can we auto wrap everything? that’d be a huge upgrade from qt

Reply:
we can wrap rl.draw_text, but you'd need to programmatically explore all parts of the UI to create/update the translations. And doesn't handle dynamic text mixed with static text